dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -541,14 +565,29 @@ def __init__(
             await _cleanup_stale_resources()
 
         self._clients = await self._get_clients()
-        self._fargate = (
-            self.config.get("fargate", False)
-            if self._fargate is None
-            else self._fargate
+        self._fargate_scheduler = (
+            self.config.get("fargate_scheduler", False)
+            if self._fargate_scheduler is None
+            else self._fargate_scheduler
+        )
+        self._fargate_workers = (
+            self.config.get("fargate_workers", False)
+            if self._fargate_workers is None
+            else self._fargate_workers
         )
         self._tags = self.config.get("tags", {}) if self._tags is None else self._tags
+        self._worker_gpu = (
+            self.config.get("worker_gpu")
+            if self._worker_gpu is None
+            else self._worker_gpu
+        )  # TODO Detect whether cluster is GPU capable
```

Comment:
How do people select which kind of GPU they want?  Is this handled in some other configuration definition somewhere?

Reply:
They do when they create the ECS cluster, which is out of scope for this library.